File: java/ABTestExample/app/src/main/java/com/pubscale/sdkone/example/abtestexample/event_listener/AppOpenAdEventListener.java
package com.pubscale.sdkone.example.abtestexample.event_listener;

public interface AppOpenAdEventListener {
    void onAdLoading();
    void onAdLoaded();
    void onAdLoadFailed();
    void onAdShowFailed();
    void onAdOpened();
    void onAdClosed();
}

File: kotlin/ABTestExample/app/src/main/java/com/pubscale/sdkone/example/abtestexample/event_listener/AppOpenAdEventListener.kt
package com.pubscale.sdkone.example.abtestexample.event_listener

interface AppOpenAdEventListener {
    fun onAdLoading()
    fun onAdLoaded()
    fun onAdLoadFailed()
    fun onAdShowFailed()
    fun onAdOpened()
    fun onAdClosed()
}